java-spiffe-core/src/main/java/io/spiffe/spiffeid/TrustDomain.java

```diff
@@ -67,10 +69,23 @@ public String toString() {
         return name;
     }
 
+    /**
+     * Returns the trust domain as SPIFFE ID string (e.g. 'spiffe://example.org')
+     *
+     * @return a String formatted as a SPIFFE ID
+     */
+    public String toIdString() {
+        return String.format("%s://%s", SpiffeId.SPIFFE_SCHEME, name);
```

Comment:
`String.format()` is much slower than concatenation with `StringBuilder` or `+` (which the compiler will transform into using `StringBuilder`). I would rather see a concatenation used here since we don't really need any specific features of `String.format()`, i.e. decimal formatting